varsome_api/models/elements/ncbi.py

```diff
@@ -0,0 +1,44 @@
+from jsonmodels import models, fields
+
+from varsome_api.models.fields import DictField
+
+__author__ = "ckopanos"
+
+
+class DbSNP(models.Base):
+    version = fields.StringField(help_text="Version")
+    rsid = fields.ListField(items_types=(int,), help_text="RS ID")
+
+
+class ClinVarDisease(models.Base):
```

Comment:
ClinVar and ClinVarDisease are not needed anymore, we only have them for old annotations for the portal, right?

Reply:
Yes correct.